src/android/InAppBrowser.java

```diff
@@ -375,6 +382,16 @@ public void run() {
             pluginResult.setKeepCallback(true);
             this.callbackContext.sendPluginResult(pluginResult);
         }
+        else if (action.equals("reload")) {
```

Comment:
why does this need a command that the javascript layer can call ? Should the reloading be fully handled by the webview itself?

Reply:
I had added it in case we want to use it later. Since we don't have an immediate use for it, I'll delete it.